src/Command/ErrorFormatter/BaselinePhpErrorFormatter.php

```diff
@@ -77,4 +80,12 @@ public function formatErrors(
 		return 1;
 	}
 
+	/**
+	 * Exports the input to a double-quoted string making sure newlines are converted to `\n`.
+	 */
+	public static function exportDoubleQuoted(string $input): string
+	{
+		return '"' . substr(str_replace([PHP_EOL, '$', '"'], ['\n', '\$', '\"'], var_export($input, true)), 1, -1) . '"';
```

Comment:
Oh god, I have no idea what's going on here, and I'm afraid to merge it. Also, it'd be much nicer to escape the thing that's going into `preg_quote` (the most-inner one) instead of the thing that's already PCRE-escaped, and also nette/di-escaped.

Reply:
I tried that, but that doesn't work. 

It's important that `$`  and `"` are escaped to be put inside a double quoted string.

If I replace before putting in `preg_quote` it escapes the escape characters.
Example: https://3v4l.org/O4JGm#v8.1.16

I think this is hard to solve. Closed this PR in favor of https://github.com/phpstan/phpstan-src/pull/2269

I will try to fix the places that produce a multiline error (like https://github.com/phpstan/phpstan-deprecation-rules)